docs/extensibility/README.md

```diff
@@ -0,0 +1,15 @@
+# Extensibility
+
+## Overview
+
+Extensibility in Busola allows you to create a UI page for your Custom Resource Defintion (CRD). You can create it at a cluster or Namespace level. To configure the UI page of your CRD, you can add navigation nodes, lists, and details pages, as well as create and edit forms. You can also add [display](display-widgets.md) and [form](form-widgets.md) to have a graphical representation of the elements.
```

Comment:
Since it's only an overview I guess we can keep it, but no strong opinion about it. Just wanted to show at the beginning that there is such an option.
And you're right there is a `widget` word missing :D